manifests/s/Samsung/EasyPrinterManager/2.00.01.24/Samsung.EasyPrinterManager.installer.yaml

```diff
@@ -0,0 +1,16 @@
+# Created using wingetcreate 0.3.0.3
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: Samsung.EasyPrinterManager
+PackageVersion: 2.00.01.24
+MinimumOSVersion: 5.1.0.0
```

Comment:
@ItzLevvie, the minimum supported version is actually Windows XP, which is 5.1.0.0 so far as I know.

Should we just add 6.1.0.0 for all packages that have earlier versions, because that's the lowest WinGet will go?

Reply:
> @ItzLevvie, the minimum supported version is actually Windows XP, which is 5.1.0.0 so far as I know.
> 
> Should we just add 6.1.0.0 for all packages that have earlier versions, because that's the lowest WinGet will go?

@ItzLevvie, sorry, this one probably got lost in the jumble of all the updates in the conversation. Genuine question by the way, I'm not trying to be flippant.

Reply:
> @ItzLevvie, the minimum supported version is actually Windows XP, which is 5.1.0.0 so far as I know.
> 
> Should we just add 6.1.0.0 for all packages that have earlier versions, because that's the lowest WinGet will go?

This field is actually more towards Appx and MSIX packages, also since WinGet don't actually work on anything below 10 we have the default value as 10.0.0.0 since below 10 is sorta redundant